### DISCUSSION TITLE
Is it possible to use execution result in notebook?

### DISCUSSION THREAD

**Discussion by dclong** [executablebooks/meta#147] [Q&A]
Suppose I have a notebook and have already run cells in it using Jupyter/Lab. Is it possible for jupyter-book to reuse results in the output cells?

**Comment by akhmerov** [comment]
It will be once executablebooks/jupyter-book#833 is implemented. Although subsequent builds don't rerun the cell anyway, and using the automated execution is more reliable since it reduces the chance that obsolete results end up in the book.

**Reply by choldgraf** [reply]
@dclong you can also just set execution to off entirely and it'll just use whatever outputs are in the notebooks I believe

```yaml
execute:
  execute_notebooks: 'off'
```

**Reply by dclong** [reply]
Thank you both very much. Turning off execution is what I'm looking for in the described situation. However, is there a command line to do so instead of using a configuration file?
